force-app/src/classes/Assertions.cls

```diff
@@ -27,18 +27,26 @@ public class Assertions {
     /**
```

Comment:
**question:** what about empty param ?

Should we add an API for that instead of using `Params.empty()`
ie: void hasBeenCalledWith();
ie: void hasBeenLastCalledWith();

Reply:
I'll do that to stay consistent regarding the API shape